plugin.xml

```diff
@@ -21,6 +21,7 @@
 
   <engines>
     <engine name="cordova" version=">=3.0.0"/>
+    <engine name="cordova-ios" version=">=4.3.0" />
```

Comment:
I ❤️  that we can do this, but can we remove this restriction to not impose 4.3.0+ on people? I guess those `config-file` lines will just be ignored on lower versions and we can update the readme to state manually adding the entitlements is only required for < 4.3.0.

WDYT?

Thanks,
Eddy